### PR TITLE
chore: moving getDefaultAppDeps into runners/utils

### PR DESCRIPTION
**Pull request by joshrrose**
fixed the last circular dep
# Description

# Testing & Simbot Run

# Checklist

-   [ ] Update 0x API documentation (gitbook) if needed (e.g. public facing API change).
-   [ ] Add tests to cover changes as needed.
-   [ ] All dependent changes (e.g. RFQ server, Gas Price Oracle) have been deployed (if any).

### CODE REVIEW

File: test/rfqt_test.ts [hunk 1]
```diff
@@ -13,7 +13,8 @@ import * as _ from 'lodash';
 import 'mocha';
 import * as request from 'supertest';
 
-import { getAppAsync, getDefaultAppDependenciesAsync } from '../src/app';
+import { getAppAsync } from '../src/app';
+import { getDefaultAppDependenciesAsync } from '../src/runners/utils';
```
Comment (SavDont) [inline test/rfqt_test.ts:17]: Do we want to also delete the require cache for `rfqt_test.ts` and `rfq_blocked_address_utils_test.ts` as well?

**Review by SavDont — CHANGES_REQUESTED** [review]
Added 1 comment.
Since this is the last of the circular deps, can we include a check within the CI to ensure we don't reintroduce any circular deps.

File: test/rfqt_test.ts (hunk 1 above)
Comment (joshrrose) [reply test/rfqt_test.ts:17]: i don't think we need it. the tests pass with the code the way it is now. so don't want to add weird stuff that isn't really required.

File: package.json [hunk 2]
```diff
@@ -34,6 +34,7 @@
         "fix": "eslint --fix src test && yarn prettier",
         "prettier": "prettier --write ${npm_package_config_prettier_target} --config .prettierrc",
         "prettier:ci": "prettier --list-different ${npm_package_config_prettier_target} --config .prettierrc",
+	"circular": "madge --circular --extensions ts ./",
```
Comment (SavDont) [inline package.json:37]: nit: can we align this with rest of the scripts?

**Review by SavDont — APPROVED** [review]
LGTM

**Pull request merged**